app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileViewModel.kt

```diff
@@ -107,6 +109,7 @@ class UserProfileViewModel @Inject constructor(
         // TODO
         viewModelScope.launch {
             dataStore.clear() // TODO this should be moved to some service that will clear all the data in the app
+            logoutUseCase()
```

Comment:
Could this bring any problems if we are calling after `dataStore.clear()`? Like could the logoutUseCase use anything from the dataStore when really logging the user out?

Reply:
The app Datastore now is used for some properties like showing suppressing warnings for changing user status, and its implementation in the app is not optimal  since it does not take multiple users into consecration.
 Back to your question now kalium needs no data from the app, it's the other way around so deleting app data before kalium data should not be an issue.